Mathlib/Analysis/Normed/Group/InfiniteSum.lean

```diff
@@ -104,9 +107,25 @@ theorem Summable.of_norm_bounded [CompleteSpace E] {f : ι → E} (g : ι → 
   rw [summable_iff_cauchySeq_finset]
   exact cauchySeq_finset_of_norm_bounded g hg h
 
+open ENNReal
+variable  {ε : Type*} [TopologicalSpace ε] [ENormedAddCommMonoid ε]
+
+theorem HasSum.enorm_le_of_bounded {f : ι → ε} {g : ι → ℝ≥0∞} {a : ε} {b : ℝ≥0∞} (hf : HasSum f a)
```

Comment:
Can you merge master, please? These changes were already merged into mathlib --- after merging, they should disappear from the diff. Thanks!